packages/build/test-d/config/netlify_config.ts

```diff
@@ -50,7 +51,8 @@ const testNetlifyConfigRedirects: OnPreBuild = function ({
   expectType<string | undefined>(redirect.signed)
   expectType<string | undefined>(redirect.query && redirect.query.testVar)
   expectType<string | undefined>(redirect.headers && redirect.headers.testVar)
-  expectType<readonly string[] | undefined>(
-    redirect.conditions && redirect.conditions.language && redirect.conditions.language,
-  )
+  expectType<readonly string[] | undefined>(redirect.conditions && redirect.conditions.Language)
+  expectType<readonly string[] | undefined>(redirect.conditions && redirect.conditions.Cookie)
+  expectType<readonly string[] | undefined>(redirect.conditions && redirect.conditions.Country)
+  expectType<readonly string[] | undefined>(redirect.conditions && redirect.conditions.Role)
```

Comment:
Would it work to put those under a `if (redirect.conditions !== undefined) { ... }` block?

Reply:
Hmm. I don't know tsd very well, but I guess it should